_posts/2021-12-13-hhast-4.139.0.markdown

```diff
@@ -0,0 +1,69 @@
+---
+title: "HHAST v4.139.0, do you want more and smarter linters?"
+layout: post
+author: atry
+category: blog
+---
+
+HHAST 4.139.0 is released! This release fixes some minor issues in the `HHClientLinter` introduced by [v4.135.0](https://github.com/hhvm/hhast/releases/tag/v4.135.0).
+
+HHVM 4.126 and above remain supported.
+
+## About HHAST
+
+HHAST is a toolkit for processing the AST of Hack files. It includes a linting framework, along with many built-in linters, serving as an LSP server working with IDE to report lint errors at real time.
```

Comment:
Explain the acronyms, or go higher level overall. For example, "for analyzing the structure of Hack files". Explain LSP server -> IDE/editor features

It also includes a 'migration' framework.

Reply:
I revised the post to avoid AST and LSP. I think IDE is common enough to be mentioned directly.